docs/admin/service-accounts-admin.md

```diff
@@ -96,13 +96,14 @@ account, and the controller will update it with a generated token:
 ```json
 secret.json:
 {
+    "apiVersion": "v1",
```

Comment:
Yeah, it's kinda odd. 
Seems all right directly from the [page](https://github.com/ZJU-SEL/kubernetes/blob/fix-serviceaccount-doc/docs/admin/service-accounts-admin.md#to-create-additional-api-tokens) via "view" button.
Could it have anything to do with the format here?